### ISSUE TITLE
ATF Fonts

### ISSUE THREAD

**Issue by cmc333333** [18F/atf-eregs#16]
Should match atf.gov

**Comment by cmc333333** [comment]
Quick and dirty:
```diff
diff --git a/regulations/static/regulations/css/less/fonts.less b/regulations/static/regulations/css/less/fonts.less
index ef94a9c..9cfe7a8 100644
--- a/regulations/static/regulations/css/less/fonts.less
+++ b/regulations/static/regulations/css/less/fonts.less
@@ -4,6 +4,9 @@
  Place all @font-face rules in fonts.less
  */

+@import url(https://fonts.googleapis.com/css?family=Slabo+13px);
+@import url(https://fonts.googleapis.com/css?family=Work+Sans);
+
 @font-face{
     font-family: "Avenir Next";
     src: url("../../eregsip/font/e9167238-3b3f-4813-a04a-a384394eed42.eot");
diff --git a/regulations/static/regulations/css/less/mixins.less b/regulations/static/regulations/css/less/mixins.less
index a68da08..26261bc 100644
--- a/regulations/static/regulations/css/less/mixins.less
+++ b/regulations/static/regulations/css/less/mixins.less
@@ -39,22 +39,22 @@ mixins.less contains any mixins or variables to be used throughout the LESS file
  ----------------------
 */

-@body-font: Georgia, serif;
+@body-font: "Work Sans",Arial,Helvetica,Verdana,Geneva,sans-serif;

 .accent {
-    font-family: "Avenir Next", Arial, sans-serif;
+    font-family: "Slabo 13px",Palatino Linotype,Georgia,Times New Roman,Times,serif;
     font-weight: normal;
     font-style: normal;
 }

 .accent-demi {
-    font-family: "Avenir Next Demi", Arial, sans-serif;
+    font-family: "Slabo 13px",Palatino Linotype,Georgia,Times New Roman,Times,serif;
     font-weight: 600;
     font-style: normal;
 }

 .accent-demi-italic {
-    font-family: "Avenir Next Demi Italic", Arial, sans-serif;
+    font-family: "Slabo 13px",Palatino Linotype,Georgia,Times New Roman,Times,serif;
     font-weight: 600;
     font-style: italic;
 }
```
<img width="1009" alt="screen shot 2015-09-11 at 6 01 00 pm" src="https://cloud.githubusercontent.com/assets/326918/9827400/d344cb5a-58af-11e5-959e-fe75406ecb02.png">

Honestly not as terrible as I was suspecting, though not great.

**Comment by cmc333333** [comment]
Resolved with #34
